src/sdk/nsfs_glacier_backend/backend.js

```diff
@@ -5,6 +5,7 @@ const nb_native = require('../../util/nb_native');
 const s3_utils = require('../../endpoint/s3/s3_utils');
 const { round_up_to_next_time_of_day } = require('../../util/time_utils');
 const dbg = require('../../util/debug_module')(__filename);
+const config = require('../../../config');
 
 class GlacierBackend {
```

Comment:
What do you think about renaming this class to be called simply Glacier?
This is now used in so many places, that I wish it was more concise...
I also think this file can move to `src/sdk/glacier.js` and `src/sdk/glacier_tapecloud.js`

Reply:
Yep, done.